cfg/Config.mk

```diff
@@ -21,6 +21,18 @@ CFG_DIR := $(ROOT_DIR)/cfg
 KERNEL_PREFIX ?= k\#
 APP_PREFIX    ?= a\#
 
+
+## Enable the following two unstable flags.
+CARGOFLAGS += -Z unstable-options
+
+## Instruct cargo that we want to build the `compiler_builtins` crate by ourselves,
+## so that we can enable the "mem" feature in our built version.
+CARGOFLAGS += -Z build-std
+
+## Enable "mem" feature so that memory utility functions (e.g. memcpy) are not
+## mangled. Rust implicitly relies on these functions.
+CARGOFLAGS += -Z build-std-features=compiler-builtins-mem
+
```

Comment:
do we need these? I was under the impression that xargo handled these things for us, and that perhaps the feature option for "compilr-builtins-mem" needs to be passed to xargo instead?